keyboards/idobao/id61/post.rules.mk

```diff
@@ -0,0 +1,8 @@
+# The Denwir D60 case has a sealed/solid undercase,
+#  this enables switching off thoes LEDs
+# Usage:  `make idobao/id61:default UNDERGLOW=off`
+
+UNDERGLOW ?= yes
+ifneq ($(strip $(UNDERGLOW)), yes)
+  OPT_DEFS += -DID61_DISABLE_UNDERGLOW
+endif
```

Comment:
I would rather this board be split into two revisions instead of requiring users to explicitly set this variable. It also is not supported by the Configurator.

Reply:
Also if you still want to have this available as a make option, the file must be named `post_rules.mk` to actually take effect.

Reply:
[at] fauxpark - I was explicitly told not to build for case features, only pcb features on my other PR's.  This is the methodology I used in the sibling board, the ID63.  Non-underglow LED should not be supported by QMK-Config an therefor I'm ok to exclude the case specific exclusion of underglow.  The case specific keymap is in the `idobao` keymap regardless.

[at] sigprof - now that was a typo :(  Fixed now.